src/pages/note/index.tsx

```diff
@@ -41,13 +41,14 @@ const Note = () => {
 
   const senderName = searchParams.get('sn');
   const senderEmail = searchParams.get('se');
+  const senderMessage = searchParams.get('sm');
 
   useEffect(() => {
     setSelectedAnswer(null);
-    if (senderEmail && senderName) {
+    if (senderEmail && senderName && senderMessage) {
```

Comment:
Since `senderMessage` is optional, if we do a strict check like this that means as far as this logic is concerned its not optional